GMC_identification.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>"(i.e., PSF)" --&gt; this may need slightly re-wording since the PSF strictly has another meaning "common image resolution (with a circular beam)"</p><p> </p><p>"flux density to brightness temperature" --&gt; is it worth having a link to a webpage explaining their difference?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>added!</p>